src/d3a/d3a_core/sim_results/bills.py

```diff
@@ -104,8 +72,7 @@ def update_cumulative_bills(self, area):
         for child in area.children:
             self.update_cumulative_bills(child)
 
-        if area.uuid not in self.cumulative_bills_results or \
-                ConstSettings.GeneralSettings.KEEP_PAST_MARKETS is True:
```

Comment:
If it is 100% deprecated, please remove it from d3a-interface too, thanks

Reply:
@spyrostz I could not deprecate it. 100% unfortunately, because a lot of integrationtest depend on setting it to True.
I suggest to open a separate story for that. Do you agree?

Reply:
Please do, thanks a lot, this is anyways outside the scope of this story.

Reply:
https://gridsingularity.atlassian.net/browse/D3ASIM-2678